api/features/views.py

```diff
@@ -747,3 +764,94 @@ def create_segment_override(
     serializer.is_valid(raise_exception=True)
     serializer.save(environment=environment, feature=feature)
     return Response(serializer.data, status=201)
+
+
+@swagger_auto_schema(
+    method="POST",
+    request_body=CreateFeatureExportSerializer(),
+    responses={201: CreateFeatureExportSerializer()},
+)
+@api_view(["POST"])
+@permission_classes([CreateFeatureExportPermissions])
+def create_feature_export(request: Request) -> Response:
+    serializer = CreateFeatureExportSerializer(data=request.data)
+    serializer.is_valid(raise_exception=True)
+
+    export_features_for_environment.delay(
+        kwargs={
+            "environment_id": serializer.validated_data["environment_id"],
+            "tag_ids": serializer.validated_data["tag_ids"],
+        }
+    )
+
+    return Response(serializer.validated_data, status=201)
+
+
+@swagger_auto_schema(
+    method="POST",
+    responses={201: FeatureImportSerializer()},
+)
+@api_view(["POST"])
+@permission_classes([FeatureImportPermissions])
+def feature_import(request: Request, environment_id: int) -> Response:
+    uploaded_file = request.FILES["file"]
+
+    if uploaded_file.size > MAX_FEATURE_IMPORT_SIZE:
+        return Response(
+            {"detail": "File is too large"},
+            status=status.HTTP_400_BAD_REQUEST,
+        )
+
+    strategy = request.query_params.get("strategy").upper()
+
+    if not strategy:
+        return Response(
+            {"detail": "A feature import strategy query param must be supplied."},
+            status=status.HTTP_400_BAD_REQUEST,
+        )
+
+    file_content = uploaded_file.read().decode("utf-8")
+    feature_import = FeatureImport.objects.create(
+        environment_id=environment_id,
+        strategy=strategy,
+        data=file_content,
+    )
+    serializer = FeatureImportSerializer(feature_import)
+    import_features_for_environment.delay(
+        kwargs={"feature_import_id": feature_import.id}
+    )
+    return Response(serializer.data, status=201)
+
+
+@api_view(["GET"])
+@permission_classes([DownloadFeatureExportPermissions])
+def download_feature_export(request: Request, feature_export_id: int) -> Response:
```

Comment:
I wasn't really sure whether to use the `swagger_auto_schema` for this endpoint or even how to do it since there isn't a serializer involved.

Reply:
Could we make use of `serializers.FileField` here again?

Reply:
Offline talked about options:
1. Look into FileField.
2. See with docstring on view to propagate the information to the api docs.

Reply:
Ok I went with as much swagger as I could find from their docs. I wrote it up like this:
```python
@swagger_auto_schema(
    method="GET",
    responses={200: "File downloaded"},
    operation_description="This endpoint is to download a feature export file from a specific environment",
)
```